scripts/exgfs_atmos_post.sh

```diff
@@ -396,16 +397,15 @@ else   ## not_anl if_stime
         SPECIALFL="${PREFIX}special.grb2"
         SPECIALFLIDX="${PREFIX}special.grb2i"
       fi
-      fhr3=${fhr}
 
       if [[ "${SENDCOM}" = "YES" ]]; then
         #       echo "$PDY$cyc$pad$fhr" > $COMOUT/${RUN}.t${cyc}z.master.control
```

Comment:
Will there be a need for this line in the future?  If so, then perhaps they should be changed to `${fhr3}` to avoid issues at a later time.  If not, then should these lines just be deleted?

Reply:
fhr3 is already the loop variable (the zero-padded one).

Reply:
Removed.